internal/cmds/gen_sentinel.go

```diff
@@ -2,6 +2,26 @@
 
 package cmds
 
+type SentinelReplicas Completed
```

Comment:
These `Completed` types are generated by `go run gen.go` from https://github.com/redis/rueidis/blob/main/hack/cmds/commands_sentinel.json.

The JSON file should be updated as well.

Reply:
Sorry, I missed that. Done.